packages/apple-pay-integration/src/apple-pay-button-initialize-options.ts

```diff
@@ -1,10 +1,18 @@
+import { BuyNowCartRequestBody } from '@bigcommerce/checkout-sdk/payment-integration-api';
 /**
  * A set of options that are required to initialize ApplePay in cart.
  *
  * When ApplePay is initialized, an ApplePay button will be inserted into the
  * DOM. When a customer clicks on it, it will trigger Apple sheet.
  */
 export default interface ApplePayButtonInitializeOptions {
+    requiresShipping?: string;
```

Comment:
It might be better to add a comment with description of this value

Reply:
added